addons/danger/functions/fnc_tacticsReinforce.sqf

```diff
@@ -22,8 +22,13 @@ params ["_unit", ["_target", []], ["_units", []], ["_delay", 300]];
 // exit on dead leader
 if (!(_unit call EFUNC(main,isAlive))) exitWith {false};
 
-// set new time
+// free garrisons
 private _group = group _unit;
+if (EGVAR(main,Loaded_WP) && {!(_unit checkAIFeature "PATH")}) then {
+    _group = [_group] call EFUNC(wp,taskReset);
```

Comment:
this may break some missions where a Mission Builders stores the Group for later use.
Task Reset should never be called by us but always by the Mission Builder.

Reply:
Hmm. Interesting notion.  Of course, the Mission Builder would also be the one to enable "group reinforcements" in the first place.  So that isn't something that just happens. 

I think the feature is important, but it does not have to rely on the nuclear hammer that is taskReset: 
1. A lesser module can be added to Reinforce which specifically frees units from Garrison and Camp. 
2. A modifier to taskReset which _does not_ create a new group.